pkg/performanceprofile/cmd/render/render.go

```diff
@@ -66,6 +66,7 @@ func render(inputDir, outputDir string) error {
 
 	// Read asset directory fileInfo
 	filePaths, err := listFiles(inputDir)
+	klog.V(4).Infof("listed files: %v", filePaths)
```

Comment:
why V(4) ?

Reply:
this is for debugging purposes. I don't see the need for that to always be log